examples/plot_fpca.py

```diff
@@ -25,7 +25,22 @@
 # to explore datasets and obtain conclusions about said dataset using this
 # technique.
 #
-# First we are going to fetch the Berkeley Growth Study data. This dataset
+# First we will introduce what FPCA is and why it is commonly used in many
+# studies. Generally, functional data is represented as functions X(t) where
+# t might be time or some other continuous variable. In theory, these 
+# functions can have an infinite number of data points (they can be treated as
+# infinite dimension vector spaces) so the goal of FPCA is to reduce the 
```

Comment:
I would expect some definition ("FPCA is ..."), like "FPCA is a dimensionality reduction method for functional data. The goal of FPCA is to reduce [...]".